server/lib/src/idm/oauth2.rs

```diff
@@ -2070,6 +2070,10 @@ impl IdmServerProxyReadTransaction<'_> {
                 }
             };
 
+            if granted_scopes.contains(OAUTH2_SCOPE_SSH_PUBLICKEYS) {
+                pii_scopes.insert(OAUTH2_SCOPE_SSH_PUBLICKEYS.to_string());
```

Comment:
Are public keys really PII? Or is this just funny naming

Reply:
IMO yes ssh publickeys are PII and we should tell the user when we plan to share them to another service.

Reply:
I disagree on the PII bit but warning them's sensible